channels/br.m3u

```diff
@@ -63,8 +61,6 @@ http://213.13.26.11:1935/live/portocanal/playlist.m3u8
 http://flash.softhost.com.br:1935/pucgoias/aovivo/live.m3u8
 #EXTINF:-1 tvg-id="RecordNewsBrazil.br" tvg-name="Record News (Brazil)" tvg-country="BR" tvg-language="" tvg-logo="" group-title="",Record News (Brazil)
 https://rede-muhler-recordnews-1-br.samsung.wurl.com/manifest/playlist.m3u8
-#EXTINF:-1 tvg-id="RecordNewsTVGuaraMA.br" tvg-name="Record News - TV Guará (MA)" tvg-country="BR" tvg-language="" tvg-logo="https://i.imgur.com/PafEXRA.png" group-title="",Record News - TV Guará (MA)
-https://59f2354c05961.streamlock.net:1443/tvguara/_definst_/tvguara/playlist.m3u8
```

Comment:
Not 24/7

Definst isn't needed : https://59f2354c05961.streamlock.net:1443/tvguara/tvguara/playlist.m3u8